core-feature-pack/src/main/resources/content/bin/common.ps1

```diff
@@ -70,6 +70,14 @@ if (-Not $javaOpts){
 	$javaOpts = Get-Java-Opts
 }
 
+Function Check-For-Java9+ {
```

Comment:
I don't know much about PowerShell but it seems a bit weird to have a `+` in a function name.